src/test/java/org/cactoos/iterable/MappedTest.java

```diff
@@ -77,4 +79,16 @@ public void string() {
             Matchers.equalTo("2, 4, 6")
         );
     }
+
+    @Test
+    public void transformsArray() {
+        new Assertion<>(
+            "Transforms an array",
+            new Mapped<>(
+                input -> new Upper(new TextOf(input)),
+                "a", "b", "c"
+            ).iterator().next(),
+            new TextIs("A")
```

Comment:
@victornoel I think this test should check all the values of array.

Reply:
@fanifieiev thanks, it is done